pkg/api/router.go

```diff
@@ -111,6 +116,30 @@ func Run(host string) {
 	beego.Router("/", &VersionPortal{}, "get:ListVersions")
 	beego.Router("/:apiVersion", &VersionPortal{}, "get:GetVersion")
 
+	// beego https config
+	beego.BConfig.Listen.EnableHTTP = false
+	beego.BConfig.Listen.EnableHTTPS = true
+	strs := strings.Split(host.ApiEndpoint, ":")
+	beego.BConfig.Listen.HTTPSAddr = strs[0]
```

Comment:
I suggest to construct `iota` variables to represent Address and Prot

Reply:
Changed.